src/main/java/model/LadderLine.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class LadderLine {
+    private ArrayList<Boolean> points = new ArrayList<>();
```

Comment:
```suggestion
    private List<Boolean> points = new ArrayList<>();
```
앞에는 List로 해주는 것이 좋습니다!